src/components/auth/torus/AuthTorus.js

```diff
@@ -45,7 +47,10 @@ import { LoginStrategy } from './sdk/strategies'
 
 const log = logger.child({ from: 'AuthTorus' })
 
-export const useAlreadySignedUp = () => {
+// eslint-disable-next-line require-await
+const useAlreadySignedUpPlaceholder = () => async () => 'signup'
+
+const _useAlreadySignedUp = () => {
```

Comment:
```suggestion
const useAlreadySignedUp = () => {
```

Reply:
Just to confirm, this is temporary to not break the ios code, as commented below